### PR TITLE
remove and re-add new service for rolled back products

### PR DESCRIPTION
**Pull request by vlewin**

### CODE REVIEW

**Comment by vlewin** [comment]
@digitaltom Should we re-install release packages in a rollback method?

File: lib/suse/connect/migration.rb [hunk 1]
```diff
@@ -23,7 +23,12 @@ def rollback(client_params = {})
           status = Status.new(config)
 
           status.installed_products.each do |product|
-            client.downgrade_product(product)
+            service = client.downgrade_product(product)
+            # INFO: Remove old product service e.g. SLES 12
+            remove_service service.name
+
+            # INFO: Add new service for the same product but with new/valid service url
```
Comment (digitaltom) [inline lib/suse/connect/migration.rb]: it's not necessarily the same service

File: lib/suse/connect/migration.rb [hunk 2]
```diff
@@ -23,7 +23,12 @@ def rollback(client_params = {})
           status = Status.new(config)
 
           status.installed_products.each do |product|
-            client.downgrade_product(product)
+            service = client.downgrade_product(product)
+            # INFO: Remove old product service e.g. SLES 12
+            remove_service service.name
```
Comment (digitaltom) [inline lib/suse/connect/migration.rb]: i think we need to use 'obsoleted_service_name' attribute here?

**Comment by digitaltom** [comment]
LGTM

File: lib/suse/connect/migration.rb (hunk 2 above)
Comment (vlewin) [inline lib/suse/connect/migration.rb]: As agreed, in this case service.name is the obsolete one

**Pull request merged**